README.md

```diff
@@ -544,6 +550,24 @@ import { setRenderPartialOnly } from '@americanexpress/one-app-ducks';
 dispatch(setRenderPartialOnly(true));
 ```
 
+##### `setRenderTextOnly`
+
+Use this action creator to render text only from a holocron module, rather than a HTML.
+
+| Argument | Type | Description |
+|---|---|---|
+| `setTextOnly` | `Boolean` | (required) set whether to return text instead of HTML |
+| `tagReplacement` | `String` | Replace html tags with the character passed to this option i.e. '/n'. Defaults to empty string. |
```

Comment:
so there can only be one replacement, not possible to have `<p>` replaced with empty string and `<h1>` replaced with `/n` ?

Reply:
Yeah that's the limitation of that strip tags library replace all of them or nothing

Reply:
also looks like it has not been maintained in the last 3 years

Reply:
I saw that, not sure if it should change much...  what I like about it is doesn't use regex. We need to balance this drawback compared to creating and maintaining a new library ourselves